intel_extension_for_transformers/llm/runtime/graph/README.md

```diff
@@ -73,7 +73,7 @@ tokenizer = AutoTokenizer.from_pretrained(model_name, trust_remote_code=True)
 inputs = tokenizer(prompt, return_tensors="pt").input_ids
 streamer = TextStreamer(tokenizer)
 
-model = AutoModelForCausalLM.from_pretrained(model_name, quantization_config=config)
+model = AutoModelForCausalLM.from_pretrained(model_name, load_in_4bit=True)
```

Comment:
Remove prior `config` definition if not used.

Reply:
thanks, updated